### PR TITLE
add body and case_report section without observations

### LINKED ISSUE #1055: Develop PHDC `clinicalInformation` builder
The clinical information section is required for building a PHDC case report. We need to construct a template data class that can be populated with data coming out of the message parser.

Acceptance Criteria:
A clinical information section can be added to a PHDC case report that minimally includes the following elements with repeatable entries:
- id 
- code
- entry with observation, code, and optional value(s)

Example:
```
	<component>
		<structuredBody>
			<component>
				<section>
					<id extension="0610000005-79CAS" assigningAuthorityName="LR"/>
					<code code="55752-0" codeSystem="2.16.840.1.113883.6.1" codeSystemName="LOINC" displayName="Clinical Information"/>
					<title>Clinical Information</title>
					<entry typeCode="COMP">
						<observation classCode="OBS" moodCode="EVN">
							<code code="INV180" codeSystem="2.16.840.1.114222.4.5.1" displayName="Investigator"/>
							<value xsi:type="II" root="2.3.3.3.322.23.34" extension="Provider-101"/>
						</observation>
					</entry>
					<entry typeCode="COMP">
						<observation classCode="OBS" moodCode="EVN">
							<code code="INV515" codeSystem="2.16.840.1.114222.4.5.1" displayName="Binational Reporting Criteria"/>
							<value xsi:type="CE" code="PHC1140" codeSystem="1.2.3.5" displayName="Exposure to suspected product from Canada or Mexico"/>
							<value xsi:type="CE" code="PHC1139" codeSystem="1.2.3.5" displayName="Has case contacts in or from Mexico or Canada"/>
							<value xsi:type="CE" code="PHC1141" codeSystem="1.2.3.5" displayName="Other situations that may require binational notification or coordination of response"/>
						</observation>
					</entry>
				</section>
			</component>
		</structuredBody>
	</component>
```

### PR DESCRIPTION
**Pull request by zdeveloper**
# PULL REQUEST

## Summary
What changes are being proposed? add PHDC body and case_report section without observations

## Related Issue
Fixes #1055, #1142

## Checklist

- [ ] If this code affects the other scrum team, have they been notified? (In Slack, as reviewers, etc.)

[//]: # (PR title: Remember to name your PR descriptively!)

### CODE REVIEW

File: containers/message-parser/app/phdc/builder.py [hunk 1]
```diff
@@ -170,6 +170,46 @@ def build_header(self):
             )
         )
 
+    def build_body(self):
+        body = ET.Element("component")
+        structured_body = ET.Element("structuredBody")
```
Comment (robertmitchellv) [inline containers/message-parser/app/phdc/builder.py:175]: technically this solves #1142 which doesn't quite work if we merely insert a `<component>` and `<structuredBody>` into the `_build_base_phdc` method like the ticket says (it causes a lot of tests to fail related to header and inserts those elements in the wrong places). 

i was in the middle of implementing something similar with a "build_body" to work alongside "build_header" before i wanted to see what you and Rob were doing to be inline when i saw this! so i think it's best continue in this direction and to have your PR also fix #1142

File: .github/workflows/lint.yaml [hunk 2]
```diff
@@ -33,7 +33,7 @@ jobs:
       - name: Install dependencies
         run: |
           pip install -U pip
-          pip install black flake8
+          pip install black==23.12.1 flake8==7.0.0
```
Comment (robertmitchellv) [inline .github/workflows/lint.yaml:36]: glad this worked!

**Review by robertmitchellv — APPROVED** [review]
i like the approach of separating a `build_header` and `build_body`. i also think we should add #1142 to the fixes; i like adding both the `<component>` and `<structuredBody>` within a `build_body` method so we can better construct parent and child relationships in the right order.

looks good Zedd 🚀

File: containers/message-parser/app/phdc/builder.py (hunk 1 above)
Comment (DanPaseltiner) [reply containers/message-parser/app/phdc/builder.py:175]: If we update #1142 to have an "empty body", then the method below that is currently `_build_case_report` can be called `build_clinical_information` and handle appending directly to `self.phdc`. Then in either the `build` method or a director class in the future we handle building the appropriate sections for a given type of case report.

**Pull request merged**